app/install/centos.md

```diff
@@ -21,6 +21,9 @@ section on the page below.
 
 - [RPM Repository](https://bintray.com/kong/kong-community-edition-rpm)
 
+Note: you need to append your OS info to `baseurl`; for instance:
+`https://kong.bintray.com/kong-community-edition-rpm/centos/$releasever`
```

Comment:
This step is very intuitive :/ I expect to receive questions/tickets about it.

Maybe we can try to make it clearer (although I am convinced some users will miss it), as:
```
Note: ensure that the `baseurl` of the generated `.repo` file contains your CentOS version; for instance:

    baseurl=https://kong.bintray.com/kong-community-edition-rpm/6
    or:
    baseurl=https://kong.bintray.com/kong-community-edition-rpm/7
```

Eventually, I think it would be better to embed these instructions in getkong.org (as discussed in the past).

Reply:
Agreed. Just made those changes.